lib/screens/AudioPlayerScreen.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
Is there not a library for this audiplayer?

Reply:
Tentap does not support ones that are from libraries